Test/TestApp/src/Main.cpp

```diff
@@ -1,21 +1,26 @@
 #include <iostream>
 
 #include <pp/PolyPlugin.hpp>
+#include <pp/Defines.hpp>
 #include <AddIntent.hpp>
-#include <windows.h>
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 int main()
 {
 	pp::PluginsContainer container;
-	char ownPth[MAX_PATH];
 
+// probably Unix approach is crossplatform
+#if defined(_WIN32)
+	char ownPth[MAX_PATH];
 	HMODULE hModule = GetModuleHandle(NULL);
 	if (hModule != NULL)
 	{
 		GetModuleFileName(hModule, ownPth, (sizeof(ownPth)));
 		container.load(std::filesystem::path(ownPth).parent_path(), false);
 	}
+#else
+	container.load(std::filesystem::current_path(), false);
```

Comment:
I wanted to point out that std::filesystem::current_path() returns path of the process not the exe file but i think this approach is good enough so just leave the unix one

Reply:
I think that for a testing application this is good enough. In a normal scenario it is handled properly.